src/content/datachannel/basic/js/test.js

```diff
@@ -1,49 +1,31 @@
 /*
- *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *  Copyright (c) 2018 The WebRTC project authors. All Rights Reserved.
  *
  *  Use of this source code is governed by a BSD-style license
  *  that can be found in the LICENSE file in the root of the source
  *  tree.
  */
-/* eslint-env node */
+export default {
+  'It should transfer text over data channel': (browser) => {
+    const path = '/src/content/datachannel/basic/index.html';
+    const url = 'file://' + process.cwd() + path;
 
-'use strict';
+    browser
+      .url(url)
+      .click('#startButton')
+      .expect.element('#sendButton').to.be.enabled.before(50);
+    browser.expect.element('#dataChannelSend').to.be.enabled.before(50);
 
-// This is a basic test file for use with testling.
-// The test script language comes from tape.
-const test = require('tape');
+    browser.setValue('#dataChannelSend', 'HELLO, WORLD!');
+    browser
+      .click('#sendButton')
+      .pause(50)
+      .assert.value('#dataChannelReceive', 'HELLO, WORLD!');
 
-// Disabling on firefox until sendKeys is fixed.
-// https://github.com/mozilla/geckodriver/issues/683
-test('Basic datachannel sample', {skip: process.env.BROWSER === 'firefox'},
-  t => {
-    const webdriver = require('selenium-webdriver');
-    const seleniumHelpers = require('webrtc-utilities').seleniumLib;
-    const driver = seleniumHelpers.buildDriver();
-    const path = '/src/content/datachannel/basic/index.html';
-    const url = (process.env.BASEURL ? process.env.BASEURL : ('file://' + process.cwd())) + path;
+    browser
+      .click('#closeButton')
+      .expect.element('#sendButton').to.not.be.enabled.before(50);
```

Comment:
this removes the webrtc specifics of this test?

Reply:
Yes. The point here is to show how to do a UI test for WebRTC. We let the WebRTC API control the enabled state and the test will just verify that.

Reply:
so how does this teach people how to test webrtc then?